Sources/CodeEditTextView/Extensions/NSTextLayoutManager+textLineFragment.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  File.swift
+//  
+//
+//  Created by Khan Winter on 7/8/23.
+//
+
+import AppKit
+
+extension NSTextLayoutManager {
```

Comment:
It looks like copy from `STTextView`.

FYI this is exposed as a public API in [STTextKitPlus](https://github.com/krzyzanowskim/STTextView/blob/d62b3a2ee9bde334f74c94209f287f4acc33d60d/Package.swift#L13) module now.

Reply:
Tbh it was, we relied on the extension. Thanks for the heads up!